### PR TITLE
Update ApplicationInsights version

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Updating to latest version of ApplicationInsights.

### CODE REVIEW

File: src/NuGetPackageVerifier/third-party.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
   "packages": {
     "Microsoft.ApplicationInsights": [ "2.4.0-beta3", "2.4.0-beta5" ],
```
Comment (pakrym) [inline src/NuGetPackageVerifier/third-party.json]: Do you have any idea who uses this directly?

Comment (JunTaoLuo) [inline src/NuGetPackageVerifier/third-party.json]: I only found one instance in our repos https://github.com/aspnet/AzureIntegration/pull/75.

Comment (pakrym) [inline src/NuGetPackageVerifier/third-party.json]: I meant `Microsoft.ApplicationInsights`.

Comment (JunTaoLuo) [inline src/NuGetPackageVerifier/third-party.json]: No one, but it's pulled in transitively via M.AI.AspNetCore. I've updated the version as well but I'm not sure if we should include this in the first place. Not too familiar with how this tool works.

**Pull request merged**